ref/general/java-se.adoc

```diff
@@ -25,7 +25,10 @@ Open Liberty currently only recommends the https://adoptopenjdk.net/index.html?v
 // Insert list of bugs 
 * https://bugs.openjdk.java.net/browse/JDK-8213202[JDK-8213202] Possible race condition in TLS 1.3 session resumption - this causes breaks in many of our regression tests when running with TLS 1.3.
 
-== See also
+=== See also
 
 https://openliberty.io/blog/2019/02/06/java-11.html[Open Liberty and Java 11]
 
+== Java SE 12
+
+Any recent Java SE release from AdoptOpenJDK, OpenJDK, or Oracle will work. Note that Java SE 12 is not a long-term supported release, with a premier support scheduled to in September 2019.
```

Comment:
Change 
"Any recent Java SE release from AdoptOpenJDK, OpenJDK, or Oracle will work. Note that Java SE 12 is not a long-term supported release, with a premier support scheduled to in September 2019."
to
"Any recent Java SE release from AdoptOpenJDK, OpenJDK, or Oracle works. Java SE 12 is not a long-term supported release."

Reply:
I'm curious about the September 2019 point that was there previously - would like @aguibert to 'sign off' on this change in case there's something that should be added. Otherwise, thanks for making the changes.

Reply:
I'd like to keep the Sept 2019 bit in there, since it gives readers a concrete idea of just how "short term" the release is.

Reply:
Okay, but what was the sentence meant to say? It looks like there's a typo at the moment. Should it say something like "with a long-term supported release due in September 2019"?

Reply:
yea, the original sentence was meant to say "scheduled to *end* in Sept 2019". I've updated the PR accordingly